code/game/objects/items/clown_items.dm

```diff
@@ -89,39 +91,47 @@
 	. = ..()
 	if(!proximity || !check_allowed_items(target))
 		return
+	var/clean_speedies = src.cleanspeed * min(user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)+0.1,1) //less scaling for soapies

```

Comment:
Only one definition of cleanspeed here, drop the src.

Reply:
```suggestion
	var/clean_speedies = cleanspeed * min(user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)+0.1,1) //less scaling for soapies
```